CHANGELOG.md

```diff
@@ -20,6 +18,18 @@
 
 - `Dialog`: Fix close button alignment in cases where css order changes during build. ([@lorgan3](https://github.com/lorgan3)) in [#2351](https://github.com/teamleadercrm/ui/pull/2351))
 
+## [16.0.5] - 2022-09-16
```

Comment:
B: something is off here

<img width="1250" alt="image" src="https://user-images.githubusercontent.com/1833617/190588715-f7b7e3ac-75e4-47b5-9307-56a9bf0043e3.png">

Reply:
no, 16.0.4. wasn't marked in changelog, so I have added it